app-crypt/tpm-emulator/tpm-emulator-0.7.5-r3.ebuild

```diff
@@ -39,18 +43,18 @@ src_configure() {
 		-DMTM_EMULATOR=$(usex mtm-emulator ON OFF)
 		-DBUILD_DEV=OFF
 	)
-	cmake-utils_src_configure
+	cmake_src_configure
 
 	use modules && ln -s "${BUILD_DIR}/config.h" tpmd_dev/linux
```

Comment:
`|| die` (which you need to put in a proper `if..fi` block)

Reply:
Done